sdk/communication/Azure.Communication.MediaComposition.Tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.MediaComposition.Models;
+
+namespace Azure.Communication.MediaComposition.Tests.samples
+{
+    /// <summary>
+    /// Basic Azure Communication.MediaComposition samples.
+    /// </summary>
+    public partial class Sample1_MediaCompositionClient
+    {
+        // TODO: Update the samples to tests when the E2E flow is working
+        private const string mediaCompositionId = "2x2Grid";
+        public async Task CreateMediaCompositionAsync()
+        {
+            var mediaCompositionClient = CreateMediaCompositionClient();
+
+            var layout = new MediaCompositionLayout()
+            {
+                Resolution = new LayoutResolution()
+                {
+                    Width = 1920,
+                    Height = 1080
+                },
+                Grid = new GridLayoutOptions(
+                    rows: 2,
+                    columns: 2,
+                    new string[2, 2] { { "jill", "jack" }, { "jane", "jerry" }
+                }),
+            };
+
+            var inputs = new Dictionary<string, MediaInput>()
+            {
+                {
+                    "jill",
+                    new MediaInput()
+                    { Participant = new ParticipantInput()
+                        { Id = new CommunicationIdentifierModel()
```

Comment:
I think Dom mentioned before to make sure this property is using the standard all other ACS services use. Have we double checked that?

Reply:
Yes, it is the standard one that all other ACS services are using. In the swagger, we link to this Model type which belong in the common 
![image](https://user-images.githubusercontent.com/19615595/169123052-9cf55e25-02a9-49e0-aa2f-d68514a80dfc.png)